tools/socktap/main_cam.cpp

```diff
@@ -144,6 +148,7 @@ int main(int argc, const char** argv)
                 for (auto& chain_path : vm["certificate-chain"].as<std::vector<std::string> >()) {
                     auto chain_certificate = security::load_certificate_from_file(chain_path);
                     chain.push_back(chain_certificate);
+                    cert_cache.insert(chain_certificate);
 
                     // Only add root certificates to trust store, so certificate requests are visible for demo purposes.
```

Comment:
Regarding the change above: Demonstrating certificate requests by using tickets signed by different AAs is a cleaner way and closer to the use of this feature in the real world.

If you want to keep this need for a request for the own AA certificate as a demo feature, I would propose that it is off by default and can be activated via a command line option.